projects/js-packages/number-formatters/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
 	"extends": "jetpack-js-tools/tsconfig.tsc.json",
-	"include": [ "src/index.ts" ],
 	"compilerOptions": {
-		"typeRoots": [ "./node_modules/@types/" ],
-		"sourceMap": false,
-		"outDir": "./dist/",
-		"target": "es2022"
-	}
+		"typeRoots": [ "./node_modules/@types/", "src/*" ],
+		"target": "ES2024",
+		"importHelpers": true, // Import helpers from `tslib` instead of inlining them
+		"customConditions": null // CommonJS build fails when customConditions are defined and this package doesn't need them anyway
```

Comment:
You'll need them as soon as you start using the package in Jetpack-the-plugin or anywhere else in the monorepo.